app/.env

```diff
@@ -0,0 +1,4 @@
+VITE_OIDC_ISSUER=https://accounts.google.com/
+VITE_OIDC_REDIRECT_URI=http://localhost:5173/auth-callback
+VITE_OIDC_CLIENT_ID=1095654336484-t7jf7g8a25i3l21vstjkgticl220v8t9.apps.googleusercontent.com
```

Comment:
You probably don't want to commit that

Reply:
I deleted those credentials on my google dev console